docs/package_templates/cmake_package/all/conanfile.py

```diff
@@ -139,9 +144,7 @@ def package(self):
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
         rmdir(self, os.path.join(self.package_folder, "share"))
-        rm(self, "*.la", os.path.join(self.package_folder, "lib"))
```

Comment:
I have never encountered a `*.la` file in a CMake project.

Reply:
good catch